### ISSUE TITLE
dreambooth3d

### ISSUE THREAD

**Issue by wlupc** [ashawkey/stable-dreamfusion#183]
The stable-dreamfusion is a great work. I wonder to know the differences between Dreamfusion and [dreambooth3d](https://dreambooth3d.github.io). Looking forward to receiving your response.

**Comment by MathieuTuli** [comment]
scanning the dreambooth3d paper, there's two main differences:

1. Using DreamBooth rather than Stable Diffusion (DreamFusion paper actually uses Imagen). This is already a feature in this repo I believe, where you can swap out any diffusion model you want.
2. Multi-staged training. They do a two-stage training where they initially train a nerf for a secondary data-augmented fine-tuning stage, where in each stage a varied-training version of the dreambooth model is used. I didn't read the whole thing but it seems they do this for multi-view consistency.

**Comment by zeng-yifei** [comment]
Yep, dreambooth3d is actually a dreamfusion pro+. It mainly focus on how to finetune a dreamfusion pipeline.
